.grabl/automation.yml

```diff
@@ -65,44 +65,39 @@ build:
         export ARTIFACT_USERNAME=$REPO_GRAKN_USERNAME
         export ARTIFACT_PASSWORD=$REPO_GRAKN_PASSWORD
         bazel run @graknlabs_dependencies//distribution/artifact:create-netrc
-        bazel test //test/integration/... --test_output=streamed
+        bazel test //test/integration/... --test_output=errors
```

Comment:
These tests are way too talkative with streamed output - the output can be 90,000 lines long.

Reply:
Right. Is that because of Grakn Core or Grakn Cluster?

Reply:
There isn't a significant difference between Core and Cluster when it comes to our BDD tests.

Most of the logging is:
- printing out every BDD step, including the backgrounds that repeat 100+ times
- printing out every exception stack trace
- printing out whatever else the server says